content/bootloader.md

```diff
@@ -90,11 +102,17 @@ Installed in UEFI mode
 support@pop-os:~$
 ```
 
-Additionally, if `bios_grub` is listed under `flags`, the system is installed in legacy BIOS mode.
+Additionally, if `esp` is not listed under `flags` for the Boot Partition, the system is installed in legacy BIOS mode.
 
 ### EFI Boot - Pop!_OS (systemd-boot)
 
-If the echo command at the beginning of this page says that the OS is installed in EFI mode **and** you are using Pop!_OS, follow this section. Please note that if you have an encrypted disk, you will need to first unlock it as described below.
+If the echo command at the beginning of this page says that the OS is installed in EFI mode **and** you are using Pop!_OS, follow this section. Please note that if you have an encrypted disk, you will need to first unlock it as described below. If you see the error below then your drive is encrypted with LUKS:
+
+```
+mount: /mnt: unknown filesystem type 'crypto_LUKS'.
+```
+
+Follow these [steps](/articles/bootloader#encrypted-disk) to decrypt the drive first.
```

Comment:
Instead of repeating bounce to the set of commands for encryption, I think we should do something like:
```
sudo cryptsetup luksDump /dev/nvme0n1p3
```
If we get output that states it's not encrypted that you would skip the steps for the manual unlocking of the drive:
```
Device /dev/nvme0n1p3 is not a valid LUKS device.
```